### PR TITLE
Refactor, find hwmon folder and get GPU details

### LINKED ISSUE #3: Get GPU Names
I added a library that in theory can get it from the vendor and device IDs, but I can't make it work. If you can figure out a nicer way to do it, or make that library work, please feel free!

### PR DESCRIPTION
**Pull request by euronay**
See my issue #10.

I've refactored a bit, splitting class into separate files and doing the loading of the GPU details from that class. 

I've also fixed #3 using the PCIIdentificationResolver library.

### CODE REVIEW

File: BetterFanController/GPU.cs [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+            var deviceInfo = PCIIdentificationDatabase.GetDevice(VendorId, DeviceId);
+
+            VendorName = deviceInfo.ParentVendor.VendorName;
+            DeviceName = deviceInfo.DeviceName;
+
+            var parentHwmonDirectory = Path.Combine(_devicePath, "hwmon");
+            if(!Directory.Exists(parentHwmonDirectory))
+            {
+                // No hwmon directory - this isn't a supported card. Should I throw an exception or something??
```
Comment (Krutonium) [inline BetterFanController/GPU.cs]: Probably would be a good idea, or at least write a "Hey, somthing's wrong" to the console output.

**Review by Krutonium — CHANGES_REQUESTED** [review]
Looks Awesome, if you could add that one thing I'll pull it in! :+1:

**Comment by Krutonium** [comment]
Oh yeah and if you could update the README to-do list as well!

File: BetterFanController/GPU.cs (hunk 1 above)
Comment (euronay) [reply BetterFanController/GPU.cs]: The problem is if there is no hwmon folder, then there isn't much that can be done with that card. I've logged it to the console for now.

**Review by Krutonium — COMMENTED** [review]
Awesome

**Pull request merged**